harness/byteConversionValues.js

```diff
@@ -354,93 +459,138 @@ var byteConversionValues = {
-      0.5,                  // 0.50000001,
-      0.6000000238418579,   // 0.6
-      0.699999988079071,    // 0.7
-      NaN,                  // undefined
-      -1,                   // -1
-      -0,                   // -0
-      -0.10000000149011612, // -0.1
-      -1.100000023841858,   // -1.1
-      NaN,                  // NaN
-      -127,                 // -127
-      -128,                 // -128
-      -32767,               // -32767
-      -32768,               // -32768
-      -2147483648,          // -2147483647
-      -2147483648,          // -2147483648
-      -255,                 // -255
-      -256,                 // -256
-      -65535,               // -65535
-      -65536,               // -65536
-      -4294967296,          // -4294967295
-      -4294967296,          // -4294967296
-      Infinity,             // Infinity
-      -Infinity,            // -Infinity
-      0
+      127,                     // 127
+      128,                     // 128
+      32767,                   // 32767
+      32768,                   // 32768
+      2147483648,              // 2147483647
+      2147483648,              // 2147483648
+      255,                     // 255
+      256,                     // 256
+      65535,                   // 65535
+      65536,                   // 65536
+      4294967296,              // 4294967295
+      4294967296,              // 4294967296
+      9007199254740992,        // 9007199254740991
+      9007199254740992,        // 9007199254740992
+      1.100000023841858,       // 1.1
+      0.10000000149011612,     // 0.1
+      0.5,                     // 0.5
+      0.5,                     // 0.50000001,
+      0.6000000238418579,      // 0.6
+      0.699999988079071,       // 0.7
+      NaN,                     // undefined
+      -1,                      // -1
+      -0,                      // -0
+      -0.10000000149011612,    // -0.1
+      -1.100000023841858,      // -1.1
+      NaN,                     // NaN
+      -127,                    // -127
+      -128,                    // -128
+      -32767,                  // -32767
+      -32768,                  // -32768
+      -2147483648,             // -2147483647
+      -2147483648,             // -2147483648
+      -255,                    // -255
+      -256,                    // -256
+      -65535,                  // -65535
+      -65536,                  // -65536
+      -4294967296,             // -4294967295
+      -4294967296,             // -4294967296
+      Infinity,                // Infinity
+      -Infinity,               // -Infinity
+      0,                       // 0
+      2049,                    // 2049
+      2051,                    // 2051
+      0.00006103515625,        // 0.00006103515625
+      0.00006097555160522461,  // 0.00006097555160522461
+      5.960464477539063e-8,    // 5.960464477539063e-8
+      2.9802322387695312e-8,   // 2.9802322387695312e-8
+      2.9802322387695312e-8,   // 2.980232238769532e-8
```

Comment:
```suggestion
      2.980232238769532e-8,   // 2.980232238769532e-8
```

Reply:
No, `Float32Array.of(2.980232238769532e-8)[0] === 2.9802322387695312e-8`.

Reply:
ah ok, the comment confused me, thanks